### ISSUE TITLE
Dataset for pre-training μ?

### ISSUE THREAD

**Issue by QiuJueqin** [XHWXD/DBSN#4]
Hi, in README you mentioned that 

> For D-BSN，we suggest to pre-train μ first by using L2 Loss, then pre-train σ_μ and σ_n by using the Pre-trained loss (./net/losses.py), finally fine-tune the whole framework.

In your [.sh script](https://github.com/XHWXD/DBSN/blob/4cf51b0c3b9ba7b0d3b2ab9e881a73d51058451c/dbsn_color/rgb_pretrain_mu.sh#L8) I noticed the dataset for pre-training μ is actually synthetic data (by adding poisson-gaussian noise to the clean images). My question is, will D-BSN be degraded or improved if use real noisy dataset to pre-train μ? 

Thanks.
